### PR TITLE
feat: read accommodations by host, added metrics

### PR DESCRIPTION
**Pull request by albertmakan**

### CODE REVIEW

File: ftrip.io.catalog-service/ftrip.io.catalog-service/Properties/launchSettings.json [hunk 1]
```diff
@@ -21,7 +21,7 @@
       "commandName": "Project",
       "launchBrowser": true,
       "launchUrl": "swagger/index.html",
-      "applicationUrl": "http://localhost:4999",
+      "applicationUrl": "http://localhost:4997",
```
Comment (panicmilos) [inline ftrip.io.catalog-service/ftrip.io.catalog-service/Properties/launchSettings.json:24]: Can you change this to 4994. This port is taken by EmailService and we cannot run both of them locally.

File: ftrip.io.catalog-service/ftrip.io.catalog-service/ftrip.io.catalog-service.csproj [hunk 2]
```diff
@@ -48,4 +48,6 @@
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
     <NoWarn>$(NoWarn);1591</NoWarn>
   </PropertyGroup>
+
+  <ProjectExtensions><VisualStudio><UserProperties properties_4launchsettings_1json__JsonSchema="" /></VisualStudio></ProjectExtensions>
```
Comment (panicmilos) [inline ftrip.io.catalog-service/ftrip.io.catalog-service/ftrip.io.catalog-service.csproj:52]: What is this 😄?

**Review by panicmilos — CHANGES_REQUESTED** [review]
Just one change in port.

**Review by panicmilos — APPROVED** [review]

File: ftrip.io.catalog-service/ftrip.io.catalog-service/ftrip.io.catalog-service.csproj (hunk 2 above)
Comment (albertmakan) [reply ftrip.io.catalog-service/ftrip.io.catalog-service/ftrip.io.catalog-service.csproj:52]: Probably a mouse slip in Visual Studio

**Pull request merged**